addons/medical_feedback/functions/fnc_effectBloodVolumeIcon.sqf

```diff
@@ -0,0 +1,44 @@
+#include "script_component.hpp"
+/*
+ * Author: 10Dozen
+ * Handles the blood volume icon
+ *
+ * Arguments:
+ * 0: Enable <BOOL>
+ * 1: Intensity 0...6 <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [true, 4] call ace_medical_feedback_fnc_effectBloodVolumeIcon
+ *
+ * Public: No
+ */
+params ["_enable", "_intensity"];
+
+private _indicatorCtrl = uiNamespace getVariable [QGVAR(BloodVolumeInfoIndicator), controlNull];
+
+if (!_enable || !GVAR(showBloodVolumeIcon) || {isNull _indicatorCtrl}) exitWith {
+    if (!isNull _indicatorCtrl && {ctrlText _indicatorCtrl != ""}) then { 
+        _indicatorCtrl ctrlSetText "";
+        _indicatorCtrl ctrlCommit 0;
```

Comment:
Don't need this for `ctrlSetText`.

Reply:
`ctrlText controlNull` returns `""`, so it will try to ctrlSetText each time (ofc there are not much chances for control to be null)

Reply:
I meant the `ctrlCommit` is not needed here.

Reply:
ctrlSetTextColor too?

Reply:
Removed ctrlCommits